src/net.cpp

```diff
@@ -2570,9 +2577,10 @@ std::vector<CAddress> CConnman::GetAddresses(CNode& requestor, size_t max_addres
 
 bool CConnman::AddNode(const std::string& strNode)
 {
+    CService resolved(LookupNumeric(strNode, Params().GetDefaultPort(strNode)));
```

Comment:
In the commit message of 12e601c33e `rpc: Prevents adding the same ip more than once when formatted differently`:

> ... formatting IPs in aborts
    different,